### PR TITLE
Documentation - Fix broken links

### PR DESCRIPTION
**Pull request by EK303**
Fixes ##11635 investigating and updating broken links

- focused only on broken links;
- checked the links manually on browser; 
- modified anchors in links and updated obsolete links, when possible; 
- github.com* links render "anchor not found" error due to instantiation on the JS side (as per the description of the issue);
- coactivate.org* domain is no longer valid, links with this domain were replaced, if the contents were found similar, otherwise, left unchanged;
- some links render 404, 403 or 400 errors, but are accessible on browser;

_Please check the following:_

-   [x] Do the tests still pass?[^1]
-   [x] Does the code comply with the style guide?
    -   [x] Run `make lint` from the Wagtail root.
-   [ ] For Python changes: Have you added tests to cover the new/fixed behaviour?
-   [ ] For front-end changes: Did you test on all of Wagtail’s supported environments?[^2]
    -   [ ] **Please list the exact browser and operating system versions you tested**:
    -   [ ] **Please list which assistive technologies [^3] you tested**:
-   [x] For new features: Has the documentation been updated accordingly?

**Please describe additional details for testing this change**.

[^1]: [Development Testing](https://docs.wagtail.org/en/latest/contributing/developing.html#testing)
[^2]: [Browser and device support](https://docs.wagtail.org/en/latest/contributing/developing.html#browser-and-device-support)
[^3]: [Accessibility Target](https://docs.wagtail.org/en/latest/contributing/developing.html#accessibility-targets)

### CODE REVIEW

File: docs/extending/custom_image_filters.md [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 Wagtail comes with [various image operations](image_tag). To add custom image operation, add `register_image_operations` hook to your `wagtail_hooks.py` file.
 
-In this example, the `willow.image` is a Pillow Image instance. If you use another image library, or like to support multiple image libraries, you need to update the filter code accordingly. See the [Willow documentation](https://willow.readthedocs.io/en/latest/index.html) for more information.
+In this example, the `willow.image` is a Pillow Image instance. If you use another image library, or like to support multiple image libraries, you need to update the filter code accordingly. See the [Willow documentation](https://willow.wagtail.org/latest/) for more information.
```
Comment (lb-) [inline docs/extending/custom_image_filters.md:7]:
```suggestion
In this example, the `willow.image` is a Pillow Image instance. If you use another image library, or like to support multiple image libraries, you need to update the filter code accordingly. See the [Willow documentation](https://willow.wagtail.org/stable/) for more information.
```

While we're changing this, let's use the stable (released) docs instead.

File: docs/contributing/index.md [hunk 2]
```diff
@@ -57,7 +57,7 @@ We welcome contributions to all aspects of Wagtail. If you would like to improve
 -   Support others with answers to questions on the [Wagtail StackOverflow topic](https://stackoverflow.com/questions/tagged/wagtail) or in Slack `#support`.
 -   Write a review of [Wagtail on G2](https://www.g2.com/products/wagtail/reviews).
 -   Provide some thoughtful feedback on the [Wagtail discussions](https://github.com/wagtail/wagtail/discussions).
--   Submit (or write) a tutorial or great package to the [This Week in Wagtail newsletter](https://wagtail.org/this-week-in-wagtail/), [Awesome Wagtail](https://github.com/springload/awesome-wagtail) or [](../advanced_topics/third_party_tutorials).
+-   Submit (or write) a tutorial or great package to the [This Week in Wagtail newsletter](https://wagtail.org/newsletter/) or [](../advanced_topics/third_party_tutorials).
```
Comment (lb-) [inline docs/contributing/index.md:60]: Please re-add the Awesome Wagtail link. 

```suggestion
-   Submit (or write) a tutorial or great package to the [This Week in Wagtail newsletter](https://wagtail.org/newsletter/), [Awesome Wagtail](https://github.com/springload/awesome-wagtail) or [](../advanced_topics/third_party_tutorials).
```

**Review by lb- — CHANGES_REQUESTED** [review]
Amazing stuff, thank you! This will help those reading the docs a lot.

Pretty great to see so many old blog posts still accessible.

Just a few minor items of feedback, let me know if you are OK to fix these up.

File: docs/advanced_topics/i18n.md [hunk 3]
```diff
@@ -353,7 +353,7 @@ Perhaps the most important bit of internationalization-related UI you can add
 to your site is a selector to allow users to switch between different
 languages.
 
-If you're not convinced that you need this, have a look at [https://www.w3.org/International/questions/qa-site-conneg#yyyshortcomings](https://www.w3.org/International/questions/qa-site-conneg#yyyshortcomings) for some rationale.
+If you're not convinced that you need this, have a look at [https://www.w3.org/International/questions/qa-site-conneg](https://www.w3.org/International/questions/qa-site-conneg) for some rationale.
```
[the remark this answers is not in the document]
Comment (EK303) [reply docs/advanced_topics/i18n.md]: Happy to help!
Sure, i've updated according to your feedback.

**Comment by EK303** [comment]
Happy to help!
Sure, i've updated according your feedback.

**Review by lb- — APPROVED** [review]
Thank you @EK303 this looks great now. 

Approved, thanks for making this contribution to Wagtail. Myself or another member of the core team will merge this in soon.

File: docs/advanced_topics/accessibility_considerations.md [hunk 4]
```diff
@@ -220,7 +220,7 @@ Occasional Wagtail users may not be aware of your site’s content guidelines, o
 
 ### Readability
 
-Readability is fundamental to accessibility. One of the ways to improve text content is to have a clear target for reading level / reading age, which can be assessed with [wagtail-readinglevel](https://github.com/vixdigital/wagtail-readinglevel) as a score displayed in rich text fields.
+Readability is fundamental to accessibility. One of the ways to improve text content is to have a clear target for reading level / reading age, which can be assessed with [wagtail-readinglevel](https://github.com/neon-jungle/wagtail-readability) as a score displayed in rich text fields.
```
Comment (laymonage) [inline docs/advanced_topics/accessibility_considerations.md:223]: wagtail-readability is a different package that uses Hallo.js. Let's point to the Torchbox fork of wagtail-readinglevel which is more up-to-date.

```suggestion
Readability is fundamental to accessibility. One of the ways to improve text content is to have a clear target for reading level / reading age, which can be assessed with [wagtail-readinglevel](https://github.com/torchbox-forks/wagtail-readinglevel) as a score displayed in rich text fields.
```

**Review by laymonage — CHANGES_REQUESTED** [review]

File: docs/advanced_topics/accessibility_considerations.md (hunk 4 above)
Comment (EK303) [reply docs/advanced_topics/accessibility_considerations.md]: updated the link

**Comment by EK303** [comment]
Thanks for feedbacks.
Always open and happy to help.

**Comment by lb-** [comment]
Thanks @EK303  - I could not find any name online for you to add to the contributors so I have just used `EK303` but let me know if you wanted a different name listed.

I have rebased your branch and added a changelog entry, will merge shortly.

Thanks for making your first contribution to Wagtail!

**Pull request merged**